students/tfbanks/Lesson03/Mailroom_pt1.py

```diff
@@ -0,0 +1,120 @@
+# mailroom_pt1.py Exercise by tfbanks
+
+# !/usr/bin/env python3
+
+# Donor Database containing names and donations
+donor_db = [
+    ['Tim Cooker', 1500.50, 25050.50, 15680.75],
+    ['Elon Musket', 5250.25, 26500],
+    ['Frank Petersmankempt', 550.60],
+    ['Megan Morgan', 650.40, 1600, 20625.40],
+    ['Marlene Wheeler', 820, 1222.80]
+]
+
+donor = []  # Craate Donor List
+for dname in donor_db:
+    d = dname[0]
+    donor.append(d)
+
+
+def thank_you():  # Code for selecting Donor and writing a Thank You Note
+    ty_donor = input('Enter the Full Name of a donor (for list of previous donors type list): ')
+
+    if ty_donor.title() == 'List':  # If List, Prints Donor List and restarts the function
```

Comment:
avoid having repetitive code like calling `.title()` every time. you can call it in place:
`ty_donor = input("").title()`